src/components/CreateSectionForm.vue

```diff
@@ -0,0 +1,241 @@
+<template>
+  <div>
+    <h2>Create/Edit Section</h2>
+    <br>
+
+    <form @submit.prevent>
+      <!-- Put fields & buttons with their labels here -->
+        <div >
+          <ValidationProvider name="Section Tag" rules="required" v-slot="{ errors }">
+            <label for="sectionTags">Section Tag</label><br>
+            <input list= "sectionTags" name="sectionTag" type="text" style="width:300px;" v-model="sectionTag">
+            <datalist id=sectionTags>
+              <!--Example, should import user's sections from db-->
+              <option value=ADAM-ECE3221-FRO1B-LEC></option>
+            </datalist>
```

Comment:
The "Section Tag" is just the name of the course. It isn't a dropdown and there aren't any choices so there is no need for the <datalist> tag.